shared/hashutil/hash_test.go

```diff
@@ -0,0 +1,17 @@
+package hashutil
+
+import (
+	"testing"
+)
+
+func TestHash(t *testing.T) {
+	hash := Hash([]byte{0})
+	test := Hash([]byte{0})
```

Comment:
Is there a more meaningful test you can add? Is there some known blake hash tests we could replicate?

Reply:
I think what I changed it to is an improvement but all I can find is: https://github.com/golang/crypto/blob/master/blake2b/blake2b_test.go

Should I modify it to this or is my modification fine?